### PR TITLE
Fix: Allow i_* tag series to be parsed as slots

### PR DESCRIPTION
**Pull request by space-pope**
This ensures we'll get a result back for a slot even if the tagger omits the leading `b_<slot>` tag.

### CODE REVIEW

File: src/main/java/io/spokestack/spokestack/nlu/tensorflow/TFNLUOutput.java [hunk 1]
```diff
@@ -64,21 +64,27 @@ public void registerSlotParsers(Map<String, SlotParser> parsers) {
...
         Map<Integer, Integer> slotLocations = new HashMap<>();
-        Integer curSlotStart = null;
+        Tuple<String, Integer> curSlot = null;
         for (int i = 0; i < tagLabels.length; i++) {
             String label = tagLabels[i];
             if (label.equals("o")) {
-                if (curSlotStart != null) {
-                    curSlotStart = null;
+                if (curSlot != null) {
+                    slotLocations.put(curSlot.second(), i);
```
Comment (noelweichbrodt) [inline src/main/java/io/spokestack/spokestack/nlu/tensorflow/TFNLUOutput.java:72]: why are you tracking `o` tags now?

Comment (space-pope) [reply src/main/java/io/spokestack/spokestack/nlu/tensorflow/TFNLUOutput.java:72]: That's not tracking `o`s; it's saying "if you're in the middle of a tag sequence when you hit an `o`, record the `o` as the end of that sequence and note that you're no longer in one".

Comment (space-pope) [reply src/main/java/io/spokestack/spokestack/nlu/tensorflow/TFNLUOutput.java:72]: (remember the half-open interval nature of sublists in Java; recording the `i` index as the end of the interval means "the sequence stops right before this index")

Comment (noelweichbrodt) [reply src/main/java/io/spokestack/spokestack/nlu/tensorflow/TFNLUOutput.java:72]: thanks for explaining

Comment (space-pope) [reply src/main/java/io/spokestack/spokestack/nlu/tensorflow/TFNLUOutput.java:72]: NP; thanks for keeping me honest.

**Pull request merged**